lib/systems/default.nix

```diff
@@ -26,7 +26,9 @@ rec {
       libc =
         /**/ if final.isDarwin then "libSystem"
         else if final.isMinGW  then "msvcrt"
-        else if final.isLinux  then "glibc"
+        else if final.isLinux && final.isGlibc then "glibc"
```

Comment:
I'm not wild about how `libc` and the new `is*` predicates are somewhat disconnected. But I see its the best choice given how things are today. Maybe the solution is to move `libc` into `parsed`?

Reply:
I'm not sure.  Related: It seems "abi" dictates the libc to be used, at least if it's not "unknown"... 

Hmm, I trust you on this but happy to give feedback on your thoughts.